src/nl/surf/eduhub_rio_mapper/relation_handler.clj

```diff
@@ -165,8 +169,9 @@
   {:pre [institution-oin (:recipient-oin rio-config)]}
   (let [mutator (fn [rel op] (-> (relation-mutation op institution-oin rel)
                                  (mutator/mutate! rio-config)))]
-    (doseq [rel missing]     (mutator rel :insert))
-    (doseq [rel superfluous] (mutator rel :delete)))
+    ;; TODO write smoketest that produces RIO error when inserting before deleting
```

Comment:
Please just make the smoketest instead of leaving a comment (for somebody else?).